scripts/release/release-new-branch.sh

```diff
@@ -70,15 +70,20 @@ git branch ${RELEASE_BRANCH} ${SHA} \
 git push upstream ${SHA}:refs/heads/${RELEASE_BRANCH} \
   || error_exit "Failed to create a remote release branch."
 
-# Update the version number and push for review.
-MASTER_BRANCH="${VERSION}-master"
-git checkout -b "${MASTER_BRANCH}" upstream/master
-echo "${NEXT_VERSION}" > ${ROOT}/VERSION
 
-git add ${ROOT}/VERSION
-git commit -m "Update version number to ${NEXT_VERSION}."
-git push --set-upstream origin "${MASTER_BRANCH}"
 
+# When NEXT_VERSION is not equal to the current one,
+# assum doing a minor release 2.x.y -> 2.{x+1}.0, so
```

Comment:
typo:  assume

so for patch release, we set next_version the same as current version in this step?  

I did not see the doc is updated for this step

Reply:
https://screenshot.googleplex.com/8DysXkfMur9V2Gz

Reply:
I am reading go/cloudesf-r

it seems it is not updated

Reply:
There is a separate section for patch release in the doc